Sources/Core/iOS/ActivityIndicatorViewObserver.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ActivityIndicatorViewObserver.swift
+//  Operations
+//
+//  Created by Matthew Holden 16/04/2016.
+//
+//
+
+import UIKit
+
+public protocol ActivityIndicatorViewAnimationInterface {
```

Comment:
Can you add some header documentation to this.

Essentially anything which is `public` needs documenting. Current documentation coverage is 70%+.